src/components/ArticleCard.jsx

```diff
@@ -1,23 +1,29 @@
 import React from 'react';
-import { Grid, Card, Image } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
 
 const ArticleCard = ({ article, index }) => {
   return (
     <div className='card-container' data-cy={`article-card-${index}`}>
-      <Grid.Column>
-        <Card>
-          <Image src='https://images.unsplash.com/photo-1487758608033-7780b34680ac?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=3589&q=80' />
-          <div className='content'>
-            <h4 data-cy='title'>{article.title}</h4>
-            <Card.Description data-cy='teaser'>
+      <Link to={`/articles/${article.id}`}>
+        <div id={article.id}>
+          <img
+            className='card-image'
+            src='https://images.unsplash.com/photo-1487758608033-7780b34680ac?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=3589&q=80'
+            alt=''
```

Comment:
why do you need  empty alt? Put something in there

Reply:
fixed